src/WebUI/Components/ServicesComponent.tsx

```diff
@@ -14,6 +14,8 @@ import { IServiceItem, IVssComponentProperties } from "../Types";
 import { ListComponent } from "./ListComponent";
 import "./ServicesComponent.scss";
 import { Utils } from "../Utils";
+import { IStatusProps, Statuses } from "azure-devops-ui/Status";
+import { PodStatusComponent } from "./PodStatusComponent";

```

Comment:
would it be good to rename this to some generic name?

Reply:
Done.